### PR TITLE
Minor docs bug-fixes

### PR DESCRIPTION
**Pull request by v-izmalkov**
Fix gh-893 feedback: Submodule box.info
Fix gh-869 Сonfusing wording in box.backup page
Fix gh-876 document reaction on error in replication configuration change
Update gh-868 Tarantool dependency list was changed 
Fix gh-915 feedback: Building from source
Fix gh-902 feedback: Сборка из исходных файлов 
Fix gh-920 feedback: Module vshard
Fix gh-922 feedback: Module vshard

### CODE REVIEW

File: doc/1.10/book/admin/backups.rst [hunk 1]
```diff
@@ -88,15 +88,14 @@ checkpoint.
 To take a mixed backup:
 
 1. Issue :ref:`box.backup.start() <admin-backups-backup_start>` on the
-   :ref:`administrative console <admin-security>`. This will suspend
-   garbage collection till the next ``box.backup.stop()`` and
-   will return a list of files to back up.
+   :ref:`administrative console <admin-security>`. This will return a list of
+   files to back up and suspend garbage collection of them till the next ``box.backup.stop()``.
```
Comment (lenkis) [inline doc/1.10/book/admin/backups.rst]: ...suspend garbage collection FOR them...

File: doc/1.10/book/admin/disaster_recovery.rst [hunk 2]
```diff
@@ -113,13 +113,12 @@ The following steps are applicable only to data in memtx storage engine.
 Your actions:
 
 1. Put all nodes in :ref:`read-only mode <cfg_basic-read_only>` and disable
-   checkpointing with :ref:`box.backup.start() <admin-backups-backup_start>`.
-   Disabling the checkpointing is
-   necessary to prevent the Tarantool garbage collector from removing files
-   made with older checkpoints.
+   deletion of expired checkpoints with :ref:`box.backup.start() <admin-backups-backup_start>`.
+   This will prevent the Tarantool garbage collector from removing files
+   made with older checkpoints until :ref:`box.backup.stop() <admin-backups-backup_start>` called.
```
Comment (lenkis) [inline doc/1.10/book/admin/disaster_recovery.rst]: ... IS called.

File: doc/1.10/book/replication/repl_add_instances.rst [hunk 3]
```diff
@@ -198,6 +198,16 @@ is less than or equal to the number of seconds specified in
 If ``replication_sync_lag`` is unset (nil) or set to TIMEOUT_INFINITY, then
 the replica skips the "sync" state and switches to "follow" immediately.
 
+In order to leave orphan mode you need to sync with enough
```
Comment (lenkis) [inline doc/1.10/book/replication/repl_add_instances.rst]: ... with A SUFFICIENT NUMBER (replication_connect_quorum) OF instances.

File: doc/1.10/book/replication/repl_add_instances.rst [hunk 4]
```diff
@@ -248,9 +258,11 @@ It is being called again in order to perform recovery.
 
     2. Connect to at least
        :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>`
-       nodes.
+       nodes. If failed - set status to 'orphan'.
+       (Attempts to sync will continue in the background and when/if they succeed
+       then 'orphan' status will end.)
```
Comment (lenkis) [inline doc/1.10/book/replication/repl_add_instances.rst]: ... 'orphan' status will be changed to 'connected'.)

File: doc/1.10/dev_guide/building_from_source.rst [hunk 5]
```diff
@@ -186,6 +190,11 @@ the preferences can differ. But strategically the steps are always the same.
 
    This creates the 'tarantool' executable in the ``src/`` directory.
 
+   .. NOTE::
+
+       If you encounter a ``curl`` or ``OpenSSL`` errors on this step try
```
Comment (lenkis) [inline doc/1.10/dev_guide/building_from_source.rst]: If you encounter ``curl`` or ``OpenSSL`` errors on this step, try
installing an ``openssl111`` package of the specific 1.1.1d version.

File: locale/ru/LC_MESSAGES/doc.po [hunk 6]
```diff
@@ -183,14 +183,14 @@ msgstr "Для создания смешанной резервной копии
 #: ../doc/1.10/book/admin/backups.rst:124
 msgid ""
 "Issue :ref:`box.backup.start() <admin-backups-backup_start>` on the "
-":ref:`administrative console <admin-security>`. This will suspend garbage"
-" collection till the next ``box.backup.stop()`` and will return a list of"
-" files to back up."
+":ref:`administrative console <admin-security>`. This will
+" return a list of files to back up and suspend garbage"
+" collection of them till the next ``box.backup.stop()``."
```
Comment (lenkis) [inline locale/ru/LC_MESSAGES/doc.po]: ... collection FOR them...

Comment (v-izmalkov) [reply locale/ru/LC_MESSAGES/doc.po]: Is it matter for translation whether i change msgid in po files if they are generated as we discussed?

**Comment by lenkis** [comment]
To be rebased and re-submitted later.

**Pull request closed without merging**